account_reconcile_restrict_partner_mismatch/readme/CREDITS.md

```diff
@@ -0,0 +1,2 @@
+The migration of this module from 13.0 to 14.0 was financially supported
```

Comment:
```suggestion
The migration of this module from 16.0 to 18.0 was financially supported
```

Reply:
This should not be changed. we need to add a new line for `The migration of this module from 16.0 to 18.0 was financially supported`